app/poros/songify_cart.rb

```diff
@@ -1,20 +1,28 @@
 class SongifyCart
-  attr_reader :repo,
-              :artist_name,
-              :artist_id
+  attr_reader :info
```

Comment:
We don't even need this attr_reader now since we never need it outside of the class.